### ISSUE TITLE
Insufficient StrBuf capacity for some errors

### ISSUE THREAD

**Issue by xentec** [DoumanAsh/json-rpc-types#5]
Hi,
a remote endpoint my app is talking with returns an error too big for `StrBuf<31>`:
```json
{"jsonrpc":"2.0","id":"1","error":{"message":"no such element elementId:19854, elementType:1","code":-7002}}
```
which results in this parse error: `Exceeds buffer capacity(31 bytes) at line 1 column 93`.

Would you consider expanding the buffer size (e.g. to 63) or switching to an inlining string type (like `smol_str` or similar)?

**Comment by DoumanAsh** [comment]
@xentec I think it would be better to make it a customization point then.
I generally consider long message as not a good approach (that is why there optional `data` member) but if it would be generic parameter you can override it

**Comment by xentec** [comment]
Optional type parameters are fine too. :smile: 

Thanks for the fast implementation, will test now.

**Comment by DoumanAsh** [comment]
I published 1.2.0
